relay-event-schema/src/protocol/span.rs

```diff
@@ -662,6 +662,30 @@ pub struct SpanData {
     #[metastructure(field = "gen_ai.operation.type", pii = "maybe")]
     pub gen_ai_operation_type: Annotated<String>,
 
+    /// The arguments for the MCP request
+    #[metastructure(field = "mcp.request.argument.*", pii = "maybe")]
+    pub mcp_request_argument: Annotated<Value>,
```

Comment:
I'm not sure if this works with the wildcard in the field name. Not sure what the solution would look like.